rinja_derive/src/config.rs

```diff
@@ -360,6 +360,26 @@ impl<'a> RawSyntax<'a> {
             }
         }
 
+        for end in [syntax.block_end, syntax.expr_end, syntax.comment_end] {
+            for prefix in ["<<", ">>", "&&", "..", "||"] {
+                if end.starts_with(prefix) {
+                    let msg = if end == prefix {
+                        format!("a closing delimiter must not start with an operator: {end:?}")
```

Comment:
nitpicking: it would be nice to know which block is impacted by this. Ie "expr_end" would be "expression block". So in this case the message would look like:

```
a closing delimiter must not start with an operator. Expression block closing delimiter is {end:?} which is also an operator
```